<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/myjpg/ubg.jpg);
</string>
  </property>
  <widget class="QPushButton" name="signupokBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/myjpg/signup.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signupclBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/myjpg/close3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="usrlineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/myjpg/2line.jpg);</string>
   </property>
   <property name="placeholderText">
    <string>Plese enter your Account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdlineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/myjpg/2line.jpg);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Please enter the Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdlineEdit2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/myjpg/2line.jpg);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Please enter the Password again</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>signupclBtn</sender>
   <signal>clicked()</signal>
   <receiver>signup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
